scripts/remote-heal/heal_for_docker_use_too_much_memory.yml

```diff
@@ -0,0 +1,88 @@
+---
+# This is the playbook to restart docker when docker use too much memory
+#
+# To run:
+#
+# 1. This playbook should be run with :
+#   ansible-playbook restart_docker_on_node.yml -e "cli_nodename=dev-preview-int-node-compute-89cf0"
+#
+- name: Ensure cli_nodename is set
+  hosts: localhost
+
+  pre_tasks:
+  - name: Ensure needed variables are set
+    fail:
+      msg: "Value {{ item }} is not set!"
+    when: "{{ item }} is undefined"
+    with_items:
+    - cli_nodename
+
+- name: Evacute the pods from the old nodes, restart docker
+  user: root
+  hosts: "{{ cli_nodename }}"
+  vars:
+    oo_clusterid:   "{{ hostvars[cli_nodename].oo_clusterid }}"
+    cluster_master: "{{ groups['oo_master_primary'] | intersect(groups['oo_clusterid_' ~ oo_clusterid]) | first }}"
+  roles:
+  - role: tools_roles/lib_openshift
```

Comment:
The path to the lib_openshift role would be wherever the openshift-ansible-roles RPM installed the lib_openshift role to (ie /usr/share/ansible/openshift-ansible/roles/) not tools_roles/lib_openshift (which doesn't exist).

Or since you are already providing the path when you call this playbook from remote-healer.py (ansible-playbook -M "/usr/share/ansible/openshift-ansible/roles/" ...), you can just put

```
- role: lib_openshift
```